tests/infra/test_configuration.py

```diff
@@ -0,0 +1,130 @@
+"""
+Unit test for EC2 configuration.
+"""
+
+import unittest
+import mock
+import io
+
+from treadmill.infra import configuration, SCRIPT_DIR
```

Comment:
We're using `pkg_resources` to set `SCRIPT_DIR` in [treadmill/__init__.py](https://github.com/Morgan-Stanley/treadmill/blob/master/treadmill/infra/__init__.py#L3) to reuse it in multiple places. Are you suggesting we directly call `pkg_resources` everywhere instead of setting `SCRIPT_DIR`?